src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/MetricsController.cs

```diff
@@ -21,11 +22,11 @@ public class MetricsController : ControllerBase
         private readonly PrometheusConfiguration _prometheusConfiguration;
 
         public MetricsController(ILogger<MetricsController> logger,
-            MetricsStoreService metricsStore,
+            IServiceProvider serviceProvider,
```

Comment:
Why was this changed?

Reply:
Unfortunately controllers must be public while I'm trying to maintain the api as internal for now. Changing one type frequently cascades to other types.